integrations/docker/images/chip-build-efr32/Dockerfile

```diff
@@ -7,7 +7,32 @@ RUN set -x \
     && DEBIAN_FRONTEND=noninteractive apt-get install -fy --no-install-recommends \
     gcc-arm-none-eabi=15:9-2019-q4-0ubuntu1 \
     binutils-arm-none-eabi=2.34-4ubuntu1+13ubuntu1 \
+    git-lfs \
+    openjdk-11-jdk \
     ccache=3.7.7-1 \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/ \
     && : # last line
+
+
+#Clone Gecko SDK 4.2.0 (d4854d2)
+RUN git clone https://github.com/SiliconLabs/gecko_sdk.git && \
+    cd ./gecko_sdk && \
+    git fetch origin && \
+    git checkout d4854d2
```

Comment:
should there be a tag for this? then maybe something like `git clone --depth=1 --branch=... ...` would result in a smaller image and shorter instructions.

Reply:
Done